lib/ts/Layout.cc

```diff
@@ -28,6 +28,12 @@
 #include "ts/ink_string.h"
 #include "ts/I_Layout.h"
 
+#include <string>
+#include <fstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
I'm -1 on `using` namespaces at this scope. At file scope use only the symbols you really want. It is OK to use a namespace within function scope.